components/vf-link/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### 2.0.1
+
+* fix: Fixed space issue to make the 'default' variant example look like a link (#1895)
+
```

Comment:
Add one more entry of doc change done by Niki 

https://github.com/visual-framework/vf-core/pull/1899
https://github.com/visual-framework/vf-core/pull/1894